utilities/casebundle-cronjob/cronjob.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: CASE-BUNDLE
+spec:
+  schedule: "0 12 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name:  CASE-BUNDLE
```

yamllint

[error] wrong indentation: expected 12 but found 10

(indentation)